src/index.js

```diff
@@ -1 +1,89 @@
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const zlib = require('zlib');
+const formidable = require('formidable');
+const { pipeline } = require('stream');
+const { writeLog } = require('./writeLog');
+
+const server = new http.Server();
+
+server.on('request', (req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const pathName = normalizedUrl.pathname.slice(1);
+
+  if (pathName === '' || pathName === 'index.html') {
```

Comment:
You can just add default value to the `pathName` - `const pathName = normalizedUrl.pathname.slice(1) || 'index.html'` and check here only one condition
```suggestion
  if (pathName === 'index.html') {
```